criprobe/cri.py

```diff
@@ -1,14 +1,52 @@
 import re
+import serial
+import serial.tools.list_ports
 
 
 class CriProbe:
 
     def __init__(self, simulated=False):
-        # Perform autodetect here
-        # When no probes are detected we'll create CR-100 and CR-250 simulated probes
-        # these simulated probes will primarily be used for unit testing
+
+        # Autodetects CRI probe/s. When no probe/s detected, creates CR-100 and CR-250 simulated probes.
```

Comment:
This comment does not match what the code actually does.  In the case where there are no probes attached self.probes will be an empty list.